data/styles/clear/include/Icons.mapcss

```diff
@@ -1032,6 +1036,8 @@ node|z16-[historic=monument],
 area|z16-[historic=monument],
 node|z16-[historic=memorial],
 area|z16-[historic=memorial],
+node|z17-[cemetery=grave],
+area|z17-[cemetery=grave],
 {icon-image: monument-m.svg;icon-min-distance: 12;}
```

Comment:
I think its better to find a new icon for graves

Reply:
Ops, better to move it together with existing landuse=cemetery, not monument.